### ISSUE TITLE
New-BcContainer fails on Docker 4.6.1 (76265)

### ISSUE THREAD

**Issue by balthaus** [microsoft/navcontainerhelper#2374]
New-BCContainer fails in creating container with error: User does not have permission to alter database 'CRONUS'...Initialization of container $containerName failed"

The windows account is an Administrator.  I tried creating with -auth UserPassword resulting in the same error.

```
$artifactUrl = Get-BcArtifactUrl -type sandbox -version 19 -select Latest -country us
New-BCContainer -accept_eula -containerName "ipay" -artifactUrl $artifactUrl -auth Windows -updateHosts -includeTestToolkit -licenseFile "C:\temp\5519174.flf"

```

```
BcContainerHelper version 3.0.3
BcContainerHelper emits usage statistics telemetry to Microsoft
BcContainerHelper is version 3.0.3
BcContainerHelper is running as administrator
Host is Microsoft Windows 11 Pro - 21H2
Docker Client Version is 20.10.13
Docker Server Version is 20.10.13
Fetching all docker images
Fetching all docker volumes
Pulling image mcr.microsoft.com/businesscentral:10.0.20348.587
10.0.20348.587: Pulling from businesscentral
8f616e6e9eec: Pulling fs layer
037d5740b404: Pulling fs layer
4536f3b269b5: Pulling fs layer
797ced9f8291: Pulling fs layer
aae5cbb0dda7: Pulling fs layer
742374d7475f: Pulling fs layer
05083a55230d: Pulling fs layer
4cc7c05d00f6: Pulling fs layer
797ced9f8291: Waiting
ac2d6d2b09da: Pulling fs layer
aae5cbb0dda7: Waiting
742374d7475f: Waiting
05083a55230d: Waiting
13b5f08b1513: Pulling fs layer
4cc7c05d00f6: Waiting
ac2d6d2b09da: Waiting
09605c0a51e6: Pulling fs layer
13b5f08b1513: Waiting
22049f38c422: Pulling fs layer
09605c0a51e6: Waiting
92963ee3b304: Pulling fs layer
22049f38c422: Waiting
e08af64da1f0: Pulling fs layer
92963ee3b304: Waiting
b24c61efaf89: Pulling fs layer
e08af64da1f0: Waiting
b24c61efaf89: Waiting
4536f3b269b5: Verifying Checksum
4536f3b269b5: Download complete
797ced9f8291: Verifying Checksum
797ced9f8291: Download complete
aae5cbb0dda7: Verifying Checksum
aae5cbb0dda7: Download complete
742374d7475f: Verifying Checksum
742374d7475f: Download complete
05083a55230d: Verifying Checksum
05083a55230d: Download complete
4cc7c05d00f6: Verifying Checksum
4cc7c05d00f6: Download complete
ac2d6d2b09da: Verifying Checksum
ac2d6d2b09da: Download complete
13b5f08b1513: Verifying Checksum
13b5f08b1513: Download complete
037d5740b404: Verifying Checksum
037d5740b404: Download complete
22049f38c422: Verifying Checksum
22049f38c422: Download complete
92963ee3b304: Verifying Checksum
92963ee3b304: Download complete
e08af64da1f0: Verifying Checksum
e08af64da1f0: Download complete
b24c61efaf89: Verifying Checksum
b24c61efaf89: Download complete
8f616e6e9eec: Download complete
09605c0a51e6: Verifying Checksum
09605c0a51e6: Download complete
8f616e6e9eec: Pull complete
037d5740b404: Pull complete
4536f3b269b5: Pull complete
797ced9f8291: Pull complete
aae5cbb0dda7: Pull complete
742374d7475f: Pull complete
05083a55230d: Pull complete
4cc7c05d00f6: Pull complete
ac2d6d2b09da: Pull complete
13b5f08b1513: Pull complete
09605c0a51e6: Pull complete
22049f38c422: Pull complete
92963ee3b304: Pull complete
e08af64da1f0: Pull complete
b24c61efaf89: Pull complete
Digest: sha256:c9ebf4f5a6d5d817aa883790f8eaac4a463c1412fc7be73573a81d7858c2e999
Status: Downloaded newer image for mcr.microsoft.com/businesscentral:10.0.20348.587
mcr.microsoft.com/businesscentral:10.0.20348.587
Using image mcr.microsoft.com/businesscentral:10.0.20348.587
Creating Container ipay
Style: sandbox
Multitenant: Yes
Version: 19.5.36567.38266
Platform: 19.0.36528.38025
Generic Tag: 1.0.2.1
Container OS Version: 10.0.20348.587 (ltsc2022)
Host OS Version: 10.0.22000.556 (21H2)
Using process isolation
Using locale en-US
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Using license file C:\temp\5519174.flf
Files in C:\ProgramData\BcContainerHelper\Extensions\ipay\my:
- AdditionalOutput.ps1
- license.flf
- MainLoop.ps1
- SetupVariables.ps1
- updatehosts.ps1
Creating container ipay from image mcr.microsoft.com/businesscentral:10.0.20348.587
987759795f7721ca25aed4f5635a08394fa1f6030fc3cd06f5c4c96d2d5ef315
Waiting for container ipay to be ready
Using artifactUrl https://bcartifacts.azureedge.net/sandbox/19.5.36567.38266/us
Using installer from C:\Run\150-new
Installing Business Central
Installing from artifacts
Starting Local SQL Server
Starting Internet Information Server
Copying Service Tier Files
c:\dl\sandbox\19.5.36567.38266\platform\ServiceTier\Program Files
c:\dl\sandbox\19.5.36567.38266\platform\ServiceTier\System64Folder
Copying PowerShell Scripts
c:\dl\sandbox\19.5.36567.38266\platform\WindowsPowerShellScripts\Cloud\NAVAdministration
c:\dl\sandbox\19.5.36567.38266\platform\WindowsPowerShellScripts\WebSearch
Copying dependencies
Copying ReportBuilder
Importing PowerShell Modules
Determining Database Collation from c:\dl\sandbox\19.5.36567.38266\us\BusinessCentral-US.bak
Initializing...
Setting host.docker.internal to 192.168.1.203 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.1.203 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.27.240.1 in container hosts file
Starting Container
Hostname is ipay
PublicDnsName is ipay
Using Windows Authentication
Starting Local SQL Server
Starting Internet Information Server
Copying Database on localhost\SQLEXPRESS from CRONUS to tenant
Taking database CRONUS offline
Warning, exception when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Waiting... - retrying
Warning, exception when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Waiting...... - retrying
Error when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Putting database CRONUS back online
Warning, exception when running: ALTER DATABASE [CRONUS] SET ONLINE
Waiting... - retrying
Warning, exception when running: ALTER DATABASE [CRONUS] SET ONLINE
Waiting...... - retrying
Error when running: ALTER DATABASE [CRONUS] SET ONLINE
User does not have permission to alter database 'CRONUS', the database does not exist, or the database is not in a state that allows access checks.
ALTER DATABASE statement failed.
at Invoke-SqlCmdWithRetry, C:\Run\HelperFunctions.ps1: line 434
at Copy-NavDatabase, C:\Run\HelperFunctions.ps1: line 203
at <ScriptBlock>, C:\Run\SetupDatabase.ps1: line 166
at <ScriptBlock>, C:\Run\navstart.ps1: line 122
at <ScriptBlock>, C:\Run\start.ps1: line 363
at <ScriptBlock>, <No file>: line 1Error
Using artifactUrl https://bcartifacts.azureedge.net/sandbox/19.5.36567.38266/us
Using installer from C:\Run\150-new
Installing Business Central
Installing from artifacts
Starting Local SQL Server
Starting Internet Information Server
Copying Service Tier Files
c:\dl\sandbox\19.5.36567.38266\platform\ServiceTier\Program Files
c:\dl\sandbox\19.5.36567.38266\platform\ServiceTier\System64Folder
Copying PowerShell Scripts
c:\dl\sandbox\19.5.36567.38266\platform\WindowsPowerShellScripts\Cloud\NAVAdministration
c:\dl\sandbox\19.5.36567.38266\platform\WindowsPowerShellScripts\WebSearch
Copying dependencies
Copying ReportBuilder
Importing PowerShell Modules
Determining Database Collation from c:\dl\sandbox\19.5.36567.38266\us\BusinessCentral-US.bak
Initializing...
Setting host.docker.internal to 192.168.1.203 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 192.168.1.203 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.27.240.1 in container hosts file
Starting Container
Hostname is ipay
PublicDnsName is ipay
Using Windows Authentication
Starting Local SQL Server
Starting Internet Information Server
Copying Database on localhost\SQLEXPRESS from CRONUS to tenant
Taking database CRONUS offline
Warning, exception when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Waiting... - retrying
Warning, exception when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Waiting...... - retrying
Error when running: ALTER DATABASE [CRONUS] SET OFFLINE WITH ROLLBACK IMMEDIATE
Putting database CRONUS back online
Warning, exception when running: ALTER DATABASE [CRONUS] SET ONLINE
Waiting... - retrying
Warning, exception when running: ALTER DATABASE [CRONUS] SET ONLINE
Waiting...... - retrying
Error when running: ALTER DATABASE [CRONUS] SET ONLINE
User does not have permission to alter database 'CRONUS', the database does not exist, or the database is not in a state that allows access checks.
ALTER DATABASE statement failed.
at Invoke-SqlCmdWithRetry, C:\Run\HelperFunctions.ps1: line 434
at Copy-NavDatabase, C:\Run\HelperFunctions.ps1: line 203
at <ScriptBlock>, C:\Run\SetupDatabase.ps1: line 166
at <ScriptBlock>, C:\Run\navstart.ps1: line 122
at <ScriptBlock>, C:\Run\start.ps1: line 363
at <ScriptBlock>, <No file>: line 1
New-BCContainer Telemetry Correlation Id: 16a17c45-b341-4ded-9669-1326be88619c
Initialization of container ipay failed
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\3.0.3\ContainerHandling\Wait-NavContainerReady.ps1:42 char:17
+ ...             throw "Initialization of container $containerName failed" ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Initialization of container ipay failed:String) [], RuntimeException
    + FullyQualifiedErrorId : Initialization of container ipay failed
 

```

**Comment by freddydk** [comment]
I have updated my docker engine version to 20.10.14 - works fine.
My assumption is that this is an antivirus problem or like.
Try hyperv isolation and see if that solves the issue?

**Comment by freddydk** [comment]
Did you resolve the porblem?

**Comment by freddydk** [comment]
Maybe try to exclude c:\programdata\bccontainerhelper from antivirus

**Comment by git-no** [comment]
Had the same issue and much more Docker issues (dead containers, could not start Docker anymore after restart of the computer. 
Did solve the issues today by deactivating Windows Defender. So I assume the reason of this issue is Windows Defender. 
Extending the exception list by the directory Freddy mentioned `c:\programdata\bccontainerhelper `and maybe in addition  the directories `docker inspect` could help (is not proven yet).

**Comment by freddydk** [comment]
@git-no - did you try HyperV isolation?
HyperV would also cause defender to not care I think.

**Comment by git-no** [comment]
@freddydk I work with **process isolation**. Did not try HyperV.

**Comment by balthaus** [comment]
to follow-up on my original problem.  Our IT company disabled Virtualization without notification, so that was easily resolvable.  I disabled Windows Defender and all went well after that.  I did keep "losing" permissions to the docker folder for some reason, but disable Defender seemed to overcome the permissions errors.

**Comment by freddydk** [comment]
Let me know whether the issue is resolved by excluding the folders - would be good to know.

Thanks

**Comment by balthaus** [comment]
i tried excluding C:\ProgramData\BcContainerHelper and C:\ProgramData\Docker from Windows Defender and received the same result.  Again turning off 'Real-time protection' works, so Defender is blocking some other resource needed.

**Comment by freddydk** [comment]
I will get in contact with the defender team and discuss what we can do

**Comment by PiotrMichalakSolteq** [comment]
Hi, I'm facing the same issue, but turning off defender and real-time protection does not help. May it be the antivirus app causing the problem?

**Comment by PiotrMichalakSolteq** [comment]
> i tried excluding C:\ProgramData\BcContainerHelper and C:\ProgramData\Docker from Windows Defender and received the same result. Again turning off 'Real-time protection' works, so Defender is blocking some other resource needed.

Here is how the event log looks:

![image](https://user-images.githubusercontent.com/92792767/165521181-543f4507-f943-45c0-92fc-0b8e2e3795e0.png)

**Comment by dieguitodrg** [comment]
Disabling Windows Defender Real Time Protection in Windows 10 worked for me

> Hi, I'm facing the same issue, but turning off defender and real-time protection does not help. May it be the antivirus app causing the problem?

After moths with this error and not using Docker in my Azure VM tried to disable Windows and worked for me

BcContainerHelper version 3.0.1
BcContainerHelper emits usage statistics telemetry to Microsoft
BcContainerHelper is version 3.0.1
BcContainerHelper is running as administrator
Host is Microsoft Windows 10 Pro - 20H2
Docker Client Version is 20.10.12
Docker Server Version is 20.10.12
Downloading application artifact /sandbox/20.0.37253.39941/es
Downloading C:\Users\diegorobles\AppData\Local\Temp\a720cf0b-ce09-460a-a48a-bfaf3330a161.zip
Unpacking application artifact to tmp folder using Expand-Archive
Downloading platform artifact /sandbox/20.0.37253.39941/platform
Downloading C:\Users\diegorobles\AppData\Local\Temp\1d49ee52-726c-4207-9d3a-7e8bf151d488.zip
Unpacking platform artifact to tmp folder using Expand-Archive
Downloading Prerequisite Components
Downloading c:\bcartifacts.cache\sandbox\20.0.37253.39941\platform\Prerequisite Components\Open XML SDK 2.5 for Microsoft Office\OpenXMLSDKv25.msi
Downloading c:\bcartifacts.cache\sandbox\20.0.37253.39941\platform\Prerequisite Components\IIS URL Rewrite Module\rewrite_2.0_rtw_x64.msi
Downloading c:\bcartifacts.cache\sandbox\20.0.37253.39941\platform\Prerequisite Components\DotNetCore\DotNetCore.1.0.4_1.1.1-WindowsHosting.exe
Fetching all docker images
Fetching all docker volumes
ArtifactUrl and ImageName specified
Building multitenant image mybcimage:sandbox-20.0.37253.39941-es-mt based on mcr.microsoft.com/businesscentral:10.0.19042.1466 with https://bcartifacts.azureedge.net/sandbox/20.0.37253.39941/
es
Pulling latest image mcr.microsoft.com/businesscentral:10.0.19042.1466
10.0.19042.1466: Pulling from businesscentral
Digest: sha256:79929bbae59837fb8a0c74b622afd7d08f066ffc14b0a4e168bde96dcbf49f92
Status: Image is up to date for mcr.microsoft.com/businesscentral:10.0.19042.1466
mcr.microsoft.com/businesscentral:10.0.19042.1466
Generic Tag: 1.0.2.1
Container OS Version: 10.0.19042.1466 (20H2)
Host OS Version: 10.0.19042.1466 (20H2)
Using process isolation
Using license file c:\flf\6961781.flf
Files in c:\bcartifacts.cache\gr5xma5x.n0k\my:
Copying Platform Artifacts
c:\bcartifacts.cache\sandbox\20.0.37253.39941\platform
Copying Database
Copying Licensefile
Copying ConfigurationPackages
C:\bcartifacts.cache\sandbox\20.0.37253.39941\es\ConfigurationPackages
Copying Extensions
C:\bcartifacts.cache\sandbox\20.0.37253.39941\es\Extensions
Copying Applications.ES
C:\bcartifacts.cache\sandbox\20.0.37253.39941\es\Applications.ES
c:\bcartifacts.cache\gr5xma5x.n0k
Sending build context to Docker daemon    2.2GB

Step 1/6 : FROM mcr.microsoft.com/businesscentral:10.0.19042.1466
 ---> eb53110e968c
Step 2/6 : ENV DatabaseServer=localhost DatabaseInstance=SQLEXPRESS DatabaseName=CRONUS IsBcSandbox=Y artifactUrl=https://bcartifacts.azureedge.net/sandbox/20.0.37253.39941/es filesOnly=False
 ---> Running in 5c4d98e6546a
Removing intermediate container 5c4d98e6546a
 ---> 3bb7ca1e9cf9
Step 3/6 : COPY my /run/
 ---> c446d663a1b8
Step 4/6 : COPY NAVDVD /NAVDVD/
 ---> 0776220ffb1a
Step 5/6 : RUN \Run\start.ps1 -installOnly -multitenant
 ---> Running in d6d8c51a3db6
Using installer from C:\Run\150-new
Installing Business Central
Installing from DVD
Starting Local SQL Server
WARNING: Waiting for service 'SQL Server (SQLEXPRESS) (MSSQL$SQLEXPRESS)' to 
start...
WARNING: Waiting for service 'SQL Server (SQLEXPRESS) (MSSQL$SQLEXPRESS)' to 
start...
Starting Internet Information Server
Copying Service Tier Files
C:\NAVDVD\ServiceTier\Program Files
C:\NAVDVD\ServiceTier\System64Folder
Copying PowerShell Scripts
C:\NAVDVD\WindowsPowerShellScripts\Cloud\NAVAdministration
C:\NAVDVD\WindowsPowerShellScripts\WebSearch
Copying dependencies
Copying ReportBuilder
Importing PowerShell Modules
Determining Database Collation
Changing Database Server Collation to Modern_Spanish_100_CS_AS
Restoring CRONUS Demo Database
Setting CompatibilityLevel for tenant on localhost\SQLEXPRESS
Exporting Application to CRONUS
Removing Application from tenant
Modifying Business Central Service Tier Config File for Docker
Creating Business Central Service Tier
Installing SIP crypto provider: 'C:\Windows\System32\NavSip.dll'
Copying Web Client Files
C:\NAVDVD\WebClient\Microsoft Dynamics NAV
Copying Client Files
C:\NAVDVD\LegacyDlls\program files\Microsoft Dynamics NAV
C:\NAVDVD\LegacyDlls\program files\Microsoft Dynamics NAV
C:\NAVDVD\LegacyDlls\systemFolder
Copying ModernDev Files
C:\NAVDVD
C:\NAVDVD\ModernDev\program files\Microsoft Dynamics NAV
Copying additional files
Copying ConfigurationPackages
C:\NAVDVD\ConfigurationPackages
Copying Test Assemblies
C:\NAVDVD\Test Assemblies
Copying Extensions
C:\NAVDVD\Extensions
Copying Applications
C:\NAVDVD\Applications
Copying Applications.ES
C:\NAVDVD\Applications.ES
Starting Business Central Service Tier
Importing CRONUS license file
Copying Database on localhost\SQLEXPRESS from tenant to default
Taking database tenant offline
Copying database files
Attaching files as new Database default
Putting database tenant back online
Mounting tenant database
Mounting Database for default on server localhost\SQLEXPRESS with AllowAppDatabaseWrite = False
Sync'ing Tenant
Tenant is Operational
Stopping Business Central Service Tier
Installation took 266 seconds
Installation complete
Removing intermediate container d6d8c51a3db6
 ---> bacc33b04f19
Step 6/6 : LABEL legal="http://go.microsoft.com/fwlink/?LinkId=837447"       created="202205020859"       nav=""       cu=""       multitenant="Y" country="ES"       version="20.0.37253.39941
"       platform="20.0.37114.39877"
 ---> Running in db53f8b38bea
Removing intermediate container db53f8b38bea
 ---> d23842e7a8c2
Successfully built d23842e7a8c2
Successfully tagged mybcimage:sandbox-20.0.37253.39941-es-mt
Building image took 767 seconds
Using image mybcimage:sandbox-20.0.37253.39941-es-mt
Creating Container bc
Style: sandbox
Multitenant: Yes
Version: 20.0.37253.39941
Platform: 20.0.37114.39877
Generic Tag: 1.0.2.1
Container OS Version: 10.0.19042.1466 (20H2)
Host OS Version: 10.0.19042.1466 (20H2)
Using process isolation
Using locale es-ES
Disabling the standard eventlog dump to container log every 2 seconds (use -dumpEventLog to enable)
Downloading C:\ProgramData\BcContainerHelper\Extensions\bc\Microsoft.VisualStudio.Services.VSIXPackage.vsix
Using license file c:\flf\6961781.flf
Files in C:\ProgramData\BcContainerHelper\Extensions\bc\my:
- AdditionalOutput.ps1
- license.flf
- MainLoop.ps1
- SetupVariables.ps1
- updatehosts.ps1
Creating container bc from image mybcimage:sandbox-20.0.37253.39941-es-mt
8f9c807c0eaee5174641c14264c9d62c40679a638476a60150b3c4668562fe6f
Waiting for container bc to be ready
Initializing...
Setting host.docker.internal to 10.0.1.5 in container hosts file (copy from host hosts file)
Setting gateway.docker.internal to 10.0.1.5 in container hosts file (copy from host hosts file)
Setting kubernetes.docker.internal to 127.0.0.1 in container hosts file (copy from host hosts file)
Setting host.containerhelper.internal to 172.21.0.1 in container hosts file
Starting Container
Hostname is bc
PublicDnsName is bc
Using NavUserPassword Authentication
Starting Local SQL Server
Starting Internet Information Server
Creating Self Signed Certificate
Self Signed Certificate Thumbprint 086A96B07B2D824E3D949D8DB68FAC9C3C75541D
DNS identity bc
Modifying Service Tier Config File with Instance Specific Settings
Starting Service Tier
Registering event sources
Creating DotNetCore Web Server Instance
Enabling Financials User Experience
Using license file 'c:\run\my\license.flf'
Import License
Dismounting Tenant
Mounting Tenant
Mounting Database for default on server localhost\SQLEXPRESS with AllowAppDatabaseWrite = False
Sync'ing Tenant
Tenant is Operational
Creating http download site
Setting SA Password and enabling SA
Creating diegorobles as SQL User and add to sysadmin
WARNING: This license is not compatible with this version of Business Central.
Creating SUPER user
WARNING: This license is not compatible with this version of Business Central.
WARNING: This license is not compatible with this version of Business Central.
Container IP Address: 172.21.2.113
Container Hostname  : bc
Container Dns Name  : bc
Web Client          : http://bc/BC/?tenant=default
Dev. Server         : http://bc
Dev. ServerInstance : BC
Dev. Server Tenant  : default
Setting bc to 172.21.2.113 in host hosts file
Setting bc-default to 172.21.2.113 in host hosts file
Setting bc-default to 172.21.2.113 in container hosts file

Files:
http://bc:8080/Microsoft.VisualStudio.Services.VSIXPackage.vsix

WARNING: You are running a container which is 109 days old.
Microsoft recommends that you always run the latest version of our containers.

Container Total Physical Memory is 16.0Gb
Container Free Physical Memory is 8.6Gb

Initialization took 73 seconds
Ready for connections!
Reading CustomSettings.config from bc
Creating Desktop Shortcuts for bc
Container bc successfully created

Use:
Get-BcContainerEventLog -containerName bc to retrieve a snapshot of the event log from the container
Get-BcContainerDebugInfo -containerName bc to get debug information about the container
Enter-BcContainer -containerName bc to open a PowerShell prompt inside the container
Remove-BcContainer -containerName bc to remove the container again
docker logs bc to retrieve information about URL's again

**Comment by PiotrMichalakSolteq** [comment]
Unfortunately switching off the Windows Defender Real Time Protection still doesent work for me. My company support even turned on F secure antivirus for me so I dont even use the Defender, but there is no info there about the malware or something trough the initialization.

**Comment by freddydk** [comment]
I have changed some things in the generic image (in a preview version), which should fix this issue.
Have a look here: https://freddysblog.com/2022/05/10/resilience/
And let me know whether that works for you.

**Comment by PiotrMichalakSolteq** [comment]
Tried solution from blog post and it works for me.

**Comment by freddydk** [comment]
Generic image 1.0.2.5 has now shipped together with BcContainerHelper 3.0.8
